api/GlobalEventHandlers.json

```diff
@@ -2806,51 +2806,6 @@
           }
         }
       },
-      "onresize": {
-        "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/GlobalEventHandlers/onresize",
-          "spec_url": "https://html.spec.whatwg.org/multipage/webappapis.html#handler-onresize",
-          "support": {
-            "chrome": {
-              "version_added": "34"
-            },
-            "chrome_android": "mirror",
-            "edge": [
-              {
-                "version_added": "79"
-              },
-              {
-                "version_added": "12",
-                "version_removed": "79",
-                "partial_implementation": true,
-                "notes": "In EdgeHTML versions of Edge, this handler was only supported on the <a href='https://developer.mozilla.org/docs/Web/API/Window'><code>Window</code></a> API."
```

Comment:
Turns out this was about missing `onresize` for `<video>` elements, and I've sent https://github.com/mdn/browser-compat-data/pull/16865 to fix this.